sessions/js-fetch/star-wars/components/Card/Card.css

```
@@ -0,0 +1,13 @@
+.card {
+	padding: 20px;
+	border-radius: 14px;
+	background-color: #eaf1f1;
+	display: grid;
+	gap: 10px;
+	box-shadow: rgba(60, 0, 50, 0.24) 0px 3px 12px;
```

```suggestion
	box-shadow: 0px 2px 2px var(--granite);
```
IIRC we've stated as part of the Modern CSS Overhaul issue that we'd like to stop using rgba()? This is also somewhat of a stylistic adjustment to make the cards more cards-y. ;)

---

Nice one, I didn't check the CSS!